src/Routing/Route/Route.php

```diff
@@ -852,12 +858,26 @@ public function staticPath(): string
     {
         $routeKey = strpos($this->template, '{');
         if ($routeKey !== false && strpos($this->template, '}') !== false) {
-            return substr($this->template, 0, $routeKey);
+            $parts = explode('/', ltrim($this->template, '/'));
```

Comment:
This additional work was required to support templates like `'/api/{/{action}/*'`.

Someone actually using such a template seems quite unlikely but our test suite has a test for `'/api/{/:action/*'`, it needed to be supported with the braced placeholder too.

Reply:
This looks like a sizable amount of code for something that could lead to ambiguous results. We could require that any `{` in URL templates require an escape character like `\{` to help clarify where the route parameters are.

Reply:
Yeah that sounds reasonable.

Reply:
I remember voicing a concern about escaping unrelated `{` characters in the text/log placeholders a long time ago. Did we do the same thing there?